src/App.jsx

```diff
@@ -1,104 +1,29 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getCommentsId(commentId) {
+  return (
+    commentsFromServer.filter(comments => comments.id === commentId) || null
+  );
+}
+
+export const posts = postsFromServer.map(post => ({
+  ...post,
+  user: getUserById(post.userId),
+  comments: getCommentsId(post.commentsId),
```

Comment:
The property `commentsId` does not exist on the `post` object. If you intend to get comments related to a post, you should ensure that the correct property is used, likely `post.id` or a similar identifier.